tests/marine/test_marine.py

```diff
@@ -1025,3 +1025,22 @@ def test_parse_fields_preserves_order(marine_instance: Marine, tcp_packet: bytes
         "udp.srcport": None,
         "tcp.srcport": "16424",
     }
+
+
+def test_parse_all_fields_int_value(tcp_packet_fields):
```

Comment:
Test all types please

Reply:
Done

Reply:
Forgot `float` :)

Reply:
From what I've seen, nearly no built-in dissector utilizes `float` values.
I would remove this value type, and make it fall back to string.